build-support/docker/centos6/Dockerfile

```diff
@@ -4,18 +4,36 @@
 # Use centos6 for compatibility with glibc 2.12.
 FROM centos:6
 
-# Install python 2.7.13, (more) modern gcc, and a JDK.
+# Install a (more) modern gcc, and a JDK.
```

Comment:
Grammar nit: comma not necessary because now you only have two things in list, not three.

Alternatively, combine next line `# Install a (more) modern gcc, newer JDK, and sqlite-devel for Python 3 bootstrap (see https://stackoverflow.com/a/39907500/2518889)`.

Reply:
8977de3 did this -- with an em dash!